src/main/resources/io/jenkins/plugins/security/scan/extension/pipeline/SecurityScanStep/help-coverity_prComment_enabled.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Coverity security testing as pull request comment. Supported values: <code>true</code> or <code>false</code>
+    Coverity security testing as pull request comment. Supported values: <code>true</code> or <code>false</code>. Requires SCM Token.
```

Comment:
The tooltip for Black Duck SCA looks perfect. We should follow the same approach in Coverity and Polaris as well.